--- setup.py	2024-11-06 15:27:07.040691101 +0200
+++ setup_patch.py	2024-11-06 16:52:26.816258118 +0200
@@ -9,8 +9,8 @@
 from setuptools import find_packages, setup
 
 PROJECT_PATH = os.path.dirname(os.path.abspath(__file__))
-_jax_version_constraints = ">=0.4.25"
-_jaxlib_version_constraints = ">=0.4.25"
+_jax_version_constraints = ">=0.4.23"
+_jaxlib_version_constraints = ">=0.4.23.*"
 
 # Find version
 for line in open(os.path.join(PROJECT_PATH, "numpyro", "version.py")):
